shared/js/background/tracker-utils.js

```diff
@@ -32,12 +32,13 @@ function isTracker (url) {
 }
 
 /**
- * Determine if the social entity should be blockedon this URL. returns True if so.
+ * Determine if the social entity should be blocked on this URL. returns True if so.
  */
 function shouldBlockSocialNetwork (entity, url) {
+    const ctpEnabled = tdsStorage.config.features?.clickToPlay.state === 'enabled'
     const domain = tldts.parse(url).domain
     const excludeData = getDomainsToExludeByNetwork()
-    return excludeData.filter(e => e.domain === domain && e.entity === entity).length === 0
+    return ctpEnabled && excludeData.filter(e => e.domain === domain && e.entity === entity).length === 0
```

Comment:
Why do we need ctpEnabled if there is clickToLoadIsActive? Should we reuse `clickToLoadIsActive` in here instead?